src/leap/bitmask/services/mail/smtpbootstrapper.py

```diff
@@ -138,19 +136,25 @@ def _start_smtp_service(self):
         client_cert_path = self._smtp_config.get_client_cert_path(
             self._userid, self._provider_config, about_to_download=True)
 
-        from leap.mail.smtp import setup_smtp_gateway
+        # XXX this should be defined in leap.mail.smtp, it's in bitmask.core
+        # right now.
+        SendmailOpts = namedtuple(
+            'SendmailOpts', ['cert', 'key', 'hostname', 'port'])
```

Comment:
yes. I want to move in block many of the things that are defined in bitmask.core.mail_services right now.

and all this bootstrapping is going to be deprecated soon all the same, being moved to bonafide.